src/App.tsx

```diff
@@ -17,18 +48,36 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                searchInputValue={searchInputValue}
+                setSearchInputValue={setSearchInputValue}
+                selectedOption={selectedOption}
+                setSelectedOption={setSelectedOption}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loader && <Loader />}
+              {
+                !loader && (
+                <TodoList
+                  todos={filteredTodos}
+                  selectedTodo={selectedTodo}
+                  setSelectedTodo={setSelectedTodo}
```

Comment:
Ensure that the naming conventions for methods and props are followed. The function `setSelectedTodo` should be named according to the conventions provided in the checklist.